lib/irslackd.js

```diff
@@ -1075,6 +1237,15 @@ class IrcUser {
     }
     this.ircToSlack.set(ircTarget, slackId);
     this.slackToIrc.set(slackId, ircTarget);
+    this.mapSlackUser(ircTarget, slackId);
+  }
+  async mapSlackUser(ircTarget, slackId) {
+    if (ircTarget.substr(0, 1) === '#') return;
+    if (this.slackUsersInfo.get(slackId)) return; // no need to populate the user, he's already cached
+    var user = await this.slackWeb.paginateCallOrThrow('users.info', 'user', {
+      user: slackId,
+    });
```

Comment:
that was the trade off ... have it cached (what _really_ changes in slack anyway?)

if you're sure you'd rather it inline instead of cached, i'll do that

Reply:
commit: 56cc3e4c472cce79e4d9b825a4e676fffa3658bc

still using cache ... but can remove if we don't like it.